.docker/docker-compose.yml

```diff
@@ -15,3 +17,11 @@ services:
       - '8081:8000'
     depends_on:
       - app
+  db:
+    image: postgres:12.1-alpine
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: articles
```

Comment:
the environment variables seem complete, can't think of anything else we need

Reply:
`articles` is also a table so a different name for `POSTGRES_DB` would help avoid confusion